### PR TITLE
Add lastfm documentation

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add the new lastfm setup method.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/92299
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/lastfm.markdown [hunk 1]
```diff
@@ -8,39 +8,13 @@ ha_release: '0.20'
 ha_domain: lastfm
 ha_platforms:
   - sensor
-ha_integration_type: integration
+ha_integration_type: service
```
Comment (gjohansson-ST) [inline source/_integrations/lastfm.markdown]:
```suggestion
ha_integration_type: integration
```

File: source/_integrations/lastfm.markdown [hunk 2]
```diff
@@ -8,39 +8,13 @@ ha_release: '0.20'
...
 ---
```
Comment (gjohansson-ST) [inline source/_integrations/lastfm.markdown:13]:
```suggestion
ha_config_flow: true
---
```

**Review by gjohansson-ST — CHANGES_REQUESTED** [review]
Please address

File: source/_integrations/lastfm.markdown [hunk 3]
```diff
@@ -9,38 +9,13 @@ ha_domain: lastfm
...
+ha_config_flow: true
 ---
 
 The `lastfm` sensor platform will allow you to see whenever a user starts scrobbling, their play count, last song played, and top song played on [Last.fm](https://www.last.fm/).
 
+{% include integrations/config_flow.md %}
+
 ## Setup
 
 To get an API key you need to create an [API account](https://www.last.fm/api/account/create).
```
Comment (gjohansson-ST) [inline source/_integrations/lastfm.markdown]: I think perhaps this should be above the config flow instructions.

Comment (joostlek) [reply source/_integrations/lastfm.markdown]: Currently it's `Request an API account at {api_account_url}.`, I'm fine with changing it in core. Do these docs also need to change?

Comment (gjohansson-ST) [reply source/_integrations/lastfm.markdown]: I just mean this sentence should go above the config flow section in the docs so users see this requirement before they click further to set it up. See [example](https://www.home-assistant.io/integrations/sensibo/#prerequisites)

**Review by gjohansson-ST — APPROVED** [review]
LGTM 👍

**Pull request merged**